src/components/ReviewList.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import ReviewCard from "./ReviewCard";
+import Loading from "./Loading";
+
+const ReviewList = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [reviews, setReviews] = useState([]);
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetch("https://northcoders-games-project.herokuapp.com/api/reviews")
```

Comment:
Get axois going please! It's so much better than fetch! You just have to replace `fetch` with `axios.get` and you're away. You can also then delete line 12.